From 08b15c826f3bd3000cfb649ac14d2c76703a5224 Mon Sep 17 00:00:00 2001
From: Jouni Malinen <jouni@qca.qualcomm.com>
Date: Tue, 7 Feb 2012 16:26:20 +0200
Subject: [PATCH 27/58] P2P: Move p2p_add_device() prototype to correct header
 file

This was supposed to be an internal API to be used only within
src/p2p/*.c.

Signed-hostap: Jouni Malinen <jouni@qca.qualcomm.com>
(cherry picked from commit 59c8500f18cd379a8c8dc788fc6f01829420fbc4)
---
 src/p2p/p2p.h   |    3 ---
 src/p2p/p2p_i.h |    2 ++
 2 files changed, 2 insertions(+), 3 deletions(-)

--- a/src/p2p/p2p.h
+++ b/src/p2p/p2p.h
@@ -1490,9 +1490,6 @@ void p2p_set_cross_connect(struct p2p_da
 
 int p2p_get_oper_freq(struct p2p_data *p2p, const u8 *iface_addr);
 
-int p2p_add_device(struct p2p_data *p2p, const u8 *addr, int freq, int level,
-		   const u8 *ies, size_t ies_len);
-
 /**
  * p2p_set_intra_bss_dist - Set intra BSS distribution
  * @p2p: P2P module context from p2p_init()
--- a/src/p2p/p2p_i.h
+++ b/src/p2p/p2p_i.h
@@ -649,6 +649,8 @@ struct p2p_device * p2p_add_dev_from_go_
 						struct p2p_message *msg);
 void p2p_add_dev_info(struct p2p_data *p2p, const u8 *addr,
 		      struct p2p_device *dev, struct p2p_message *msg);
+int p2p_add_device(struct p2p_data *p2p, const u8 *addr, int freq, int level,
+		   const u8 *ies, size_t ies_len);
 struct p2p_device * p2p_get_device(struct p2p_data *p2p, const u8 *addr);
 struct p2p_device * p2p_get_device_interface(struct p2p_data *p2p,
 					     const u8 *addr);
